docs/product/accounts/migration/index.mdx

```diff
@@ -16,65 +16,72 @@ For additional reading on considering SaaS, take a look at:
 
-The export migrates your account's settings and configurations, including:
+![Changing the org slug on the settings page](./img/org-slug.png)
 
-- Project settings
-- Alert rules
-- Enabled data scrubbers
-- Team settings
-- Saved search queries
-- Integration platform metadata (Sentry 10)
+### 3. Follow the relocation instructions
 
-### 4. Send it to us
+The remaining pages in the relocation instructions will contain specific details on how to export your self-hosted instance's relocatable data. This will require terminal access to the machine running your self-hosted instance. In brief, you'll save a public key that we generate for you on your self-hosted machine, which you'll then use to execute a command that we provide to generate an encrypted tarball containing your data. You'll then upload this tarball, and we'll take it from there!
 
-To kick off the migration process, contact us using the form at [Switch to Hosted Sentry](https://sentry.io/from/self-hosted/). Once we get your request, one of our support reps will reach out through a support ticket to get the following details:
+Most relocations complete in under an hour, though certain large or complex cases can take a bit longer. If you don't get an update from us within 24 hours, please [contact support](https://sentry.io/contact/enterprise/) for further assistance.
+
+### 4. Choose a plan
+
+Once your relocation succeeds, you can take a look at our [plans](https://sentry.io/pricing/) and select one that suits your needs based on the different features and capabilities. Note that plan pricing varies based on the type of _Contract_ and _Monthly Event Volume_, so make sure you are familiar with your current/expected monthly usage (number of events per month).
+
+If you're expecting higher volumes or you're interested in our Enterprise capabilities (priority support and SLAs, and single-tenant options, for example), please [contact us](https://sentry.io/contact/enterprise/) regarding our **Enterprise Plan**.
 
-- Your **Organization slug**.
-- **Selected plan**. If you're unsure, let us know. We'll be happy to help you make the right decision. Also, if you think you'll need an Enterprise plan, we can connect you with the right person to discuss details.
-- The **sentry_export.json** file.
+### 5. Update your DSNs
 
-Alternatively, you can send us an email at [support@sentry.io](mailto:support@sentry.io) with the subject: `Self-Hosted Migration Application`.
+Though your project data has been fully relocated, the DSNs for all of your projects will have changed, as they are now located at https://sentry.io rather than your prior self-hosted domain name. This means that you'll need to update the [DSN](/product/sentry-basics/dsn-explainer/) you use in each of your SDKs for each relevant project. Once you complete this step, you'll should start to see issues and events (though hopefully not too many!) appearing as before.
```

Comment:
```suggestion
After the move, your project data will be fully relocated, but the DSNs for all of your projects will now be located in https://sentry.io instead of your prior self-hosted domain name. This means that you'll need to update the [DSN](/product/sentry-basics/dsn-explainer/) you use in each of your SDKs for each relevant project. Once you complete this step, you should start seeing issues and events appearing as before.
```